cloudify_agent/installer/config/agent_config.py

```diff
@@ -516,3 +513,10 @@ def update_agent_runtime_properties(cloudify_agent):
         cloudify_agent.pop(item, None)
     ctx.instance.runtime_properties['cloudify_agent'] = cloudify_agent
     ctx.instance.update()
+
+
+def get_windows_basedir():
+    # TODO: Get the program files directory from the machine itself
```

Comment:
A wise developer once told me not to write `TODO` in my code, but rather open a JIRA about it. Which is to say, do we also have a JIRA about it?

Reply:
This TODO already existed but yes, we should probably have a jira for it. I'll make a note to add one.